Nos/Views/Components/Author/AuthorListView.swift

```diff
@@ -36,6 +41,19 @@ struct AuthorListView: View {
         .nosNavigationBar(title: .localizable.mention)
         .onAppear {
             isSearching = true
+
+            guard let threadAuthors = threadAuthors else { return }
+            filteredAuthors = threadAuthors
+        }
+        .onChange(of: searchController.authorResults) { _, newValue in
+            filteredAuthors = []
+
+            guard let currentAuthor = currentUser.author else { return }
```

Comment:
I'm concerned about returning here with an empty filteredAuthors array. I understand that if we don't have a currentUser.author, we can't sort (and maybe everything is just broken) but I think we could set `filteredAuthors = newValue`, right?

Reply:
@joshuatbrown, can the app ever work without a currentUser? Don't I need to login before even getting to the feed?

Reply:
Yeah, I think we do need a currentUser for the app to work. Probably not a big deal here; I wonder if we could somehow make this more clear. I guess the real issue I have is that currentUser.author returns an optional. Not sure that makes sense, but it's also outside the scope of this. Still, I'd set the filteredAuthors to the expected value `filteredAuthors = newValue` before returning here.